### DISCUSSION TITLE
Is YaLM supportable?

### DISCUSSION THREAD

**Discussion by patrakov** [ggml-org/llama.cpp#936] [Ideas]
Yandex has been using YaLM neural networks in its voice assistant, Alice and its search engine Yandex Search. YaLM 100B has been released under the Apache 2.0 license, which permits research and commercial use. They have a repository with the download script here: https://github.com/yandex/YaLM-100B. Is this something that can be converted, quantized, and, given enough RAM, used with llama.cpp?

**Comment by alexander-potemkin** [comment]
@patrakov , I wonder if you got any further with that?

**Reply by patrakov** [reply]
I have not even tried.
